vllm/model_executor/model_loader.py

```diff
@@ -21,6 +21,15 @@ def _set_default_torch_dtype(dtype: torch.dtype):
     torch.set_default_dtype(old_dtype)
 
 
+@contextlib.contextmanager
+def _set_default_torch_device(device: torch.device):
```

Comment:
A dumb question: How is this different from `with torch.device()`?

Reply:
they are same, reverted